fabric-tool-attribute-api-v1/src/main/java/net/fabricmc/fabric/api/tool/attribute/v1/DynamicAttributeTool.java

```diff
@@ -33,22 +36,73 @@
 	/**
 	 * Determines the mining level of the passed stack, which is used for calculating what blocks this tool is allowed to break.
 	 *
-	 * @param stack The stack to check on.
-	 * @param user The current user of the tool, or null if there isn't any.
+	 * @param stack The stack mining the block
+	 * @param user  The current user of the tool, or null if there isn't any
```

Comment:
```suggestion
	 * @param user The current user of the tool, or null if there isn't any
```

Reply:
![image](https://user-images.githubusercontent.com/34910653/80680301-14dcfd80-8af1-11ea-9675-126fe48bf321.png)
we have been adding spaces, **and** not adding spaces at the same time, do we want to bikeshed javadoc spacing